zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/GitNotebookRepo.java

```diff
@@ -61,28 +61,34 @@ public GitNotebookRepo(ZeppelinConfiguration conf) throws IOException {
       localRepo.create();
     }
     git = new Git(localRepo);
-    maybeAddAndCommit(".");
+    checkpoint(".", "initialization commit");
```

Comment:
right, that's more friendly. Actually I changed it not to commit even at first start when initializing Git repo. So on first start all the notebooks will be unstaged, and user will explicitly commit whenever needed.